File: 프로그래머스/1/12930. 이상한 문자 만들기/이상한 문자 만들기.py
def solution(s):
    answer = []
    for i in s.split(' '):
        a=''
        for j in range(len(i)):
            if j%2==0:
                a+=i[j].upper()
            else:
                a+=i[j].lower()
        answer.append(a)
    return ' '.join(answer)

File: 프로그래머스/1/12930. 이상한 문자 만들기/이상한 문자 만들기.js
function solution(s) {
    let arr = s.split(' ').map(i=>i.toLowerCase())
    let answer = ''
    for(a of arr){
      for(let i=0; i<a.length;i++){
          if(i%2==0){answer+=a[i].toUpperCase()}else{answer+=a[i]}
      }
        answer +=' '
    }
    answer = answer.slice(0,-1)
        
    return answer
}